libraries/botbuilder-dialogs-declarative/tests/schemaMergeTest.js

```diff
@@ -3,32 +3,21 @@ const path = require('path');
 const assert = require('assert');
 const fs = require('fs');
 const Validator = require('jsonschema').Validator;
+const util = require('util');
 // eslint-disable-next-line security/detect-child-process
-const { exec } = require('child_process');
+const exec = util.promisify(require('child_process').exec);
 
 // Note: This file is intentionally not named *.test.js to ensure it isn't run
 // via `yarn run test` or `npm run test`.
 
-const maybeIt = process.env.FORCE_SCHEMA_MERGE || process.platform === 'win32' ? it : it.skip;
```

Comment:
Note: Removed to allow running on other platforms. Tested on Windows (cmd, pwsh, bash) and Ubuntu (bash).